test.sh

```diff
@@ -58,6 +58,23 @@ cd $REPO_ROOT_DIR
 
 # Get a CCM cluster if not already configured (see available settings in dcos-commons/tools/README.md):
 if [ -z "$CLUSTER_URL" ]; then
+    if [ "$SECURITY" = "strict" ]; then
```

Comment:
This is a 10 line refactor.  It's trivial.  It's not going to block the smoke tests, which have already been failing for a week.

But I won't block this further, so long as you fix it right after.